src/plugins/mer/meremulatordevicewidget.ui

```diff
@@ -322,6 +307,34 @@
      </item>
     </layout>
    </item>
+   <item row="11" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Profile:</string>
```

Comment:
We should say "Device model" to be consistent with other places, and we should also add the "Orientation" and "View mode" fields. I would also prefer to split this addition to meremulatordevicewidget out to a separate commit "Allow to set emulator mode in Options".

Reply:
Where to place this commit: before "Add options for user defined device model profiles" commit or after?

Reply:
1. I would say after, because you need the combo box based class added with that commit, right?
2. Yes, that would be enough. Do not forget about the multiple emulators case - I do not remember if the dialog is parametrized with emulator device or if it automatically works on the device used by the current project configuration, so this is something you need to take care of in this case.

Reply:
Added button called Emulator mode dialog.